packages/components/src/internal/components/entities/actions.ts

```diff
@@ -828,9 +827,10 @@ export function moveEntities(
     selectionKey?: string,
     useSnapshotSelection?: boolean,
     userComment?: string
-): Promise<MoveEntitiesResult> {
+): Promise<Query.MoveRowsResponse> {
```

Comment:
nit: You probably thought about it already but `moveEntities` feels like a pretty generic wrapper around `moveRows`. Could change to extend `Query.MoveRowsOptions` instead and convert these into options rather than a lot of different function parameters.

```ts
export interface MoveEntitiesOptions extends Omit<Query.MoveRowsOptions, 'rows' | 'success' | 'failure' | 'scope'> {
    entityDataType: EntityDataType;
}

export function moveEntities(options: MoveEntitiesOptions): Promise<Query.MoveRowsResponse> {
    // ...
}
```

Reply:
Good suggestion. That looks much cleaner.